newbank/server/NewBank.java

```diff
@@ -65,7 +67,8 @@ public synchronized String processRequest(CustomerID customer, String request) {
 	}
 
 	private String showMyAccounts(CustomerID customer) {
-		return (customers.get(customer.getKey())).accountsToString();
+		//return (customers.get(customer.getKey())).accountsToString();
+		return "Current balance: " + String.valueOf(account.get(customer.getKey()));
```

Comment:
Using the account HashMap instead of the arraylist in customers will create differences with what is being used in payThirdParty. Which should we keep?

Reply:
We can leave it as it is for the proof on concept, probably should be changed in the future

Reply:
I agree